Dockerfile

```diff
@@ -1,68 +1,79 @@
+    LOGGING_LEVEL="" \
+    CLASSPATH="" \
+    JAVA_OPTS=""
+
 LABEL name="hazelcast/hazelcast-enterprise-openshift-rhel" \
       vendor="Hazelcast, Inc." \
       version="8.1" \
-      architecture="x86_64" \
       release="${HZ_VERSION}" \
       url="http://www.hazelcast.com" \
       summary="Hazelcast Openshift Image, certified to RHEL 8" \
       description="Starts a standalone Hazelcast server instance to form a cluster based on kubernetes discovery inside Openshift" \
       io.k8s.description="Starts a standalone Hazelcast server instance to form a cluster based on kubernetes discovery inside Openshift" \
       io.k8s.display-name="Hazelcast" \
       io.openshift.expose-services="5701:tcp" \
-      io.openshift.tags="hazelcast,java8,kubernetes,rhel8"
+      io.openshift.tags="hazelcast,kubernetes,rhel8"
 
-COPY *.yaml *.sh $HZ_HOME/
-COPY licenses /licenses
-COPY mvnw $HZ_HOME/mvnw
+EXPOSE 5701
 
-### Atomic Help File
-COPY description.md /tmp/
+COPY *.sh *.yaml *.jar *.properties ${HZ_HOME}/
+COPY licenses /licenses
+COPY help.1 /help.1
 
-RUN mkdir -p "$HZ_HOME" "$HZ_CP_MOUNT" && \
-### Disable subscription-manager plugin to prevent redundant logs
-    sed -i 's/^enabled=.*/enabled=0/g' /etc/dnf/plugins/subscription-manager.conf && \
-    dnf config-manager --disable && \
-    dnf update -y  && rm -rf /var/cache/dnf && \
-    dnf -y update-minimal --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs && \
-### Add your package needs to this installation line
-    dnf -y --setopt=tsflags=nodocs install java-1.8.0-openjdk-devel apr openssl &> /dev/null && \
-### Install go-md2man to help markdown to man conversion
-    dnf -y --setopt=tsflags=nodocs install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm &> /dev/null && \
-    dnf -y --setopt=tsflags=nodocs install golang-github-cpuguy83-go-md2man &> /dev/null && \
-    go-md2man -in /tmp/description.md -out /help.1 && \
-    dnf -y remove golang-github-cpuguy83-go-md2man && \
-    dnf -y clean all && \
-    useradd -l -u $USER_UID -r -g 0 -d $HZ_HOME -s /sbin/nologin -c "${USER_UID} application user" $USER_NAME && \
-    chown -R $USER_UID:0 $HZ_HOME $HZ_CP_MOUNT && \
-    chmod -R g=u "$HZ_HOME" && \
-    chmod -R +r $HZ_HOME && \
-    cd "$HZ_HOME/mvnw" && \
-    ./mvnw -B -f dependency-copy.xml dependency:copy-dependencies && \
-    rm -rf "$HZ_HOME/mvnw" "$HZ_HOME/.m2"
+RUN mkdir -p "${HZ_HOME}/lib" "$HZ_CP_MOUNT" \
+    && echo "Disabling subscription-manager plugin to prevent redundant logs" \
+    && sed -i 's/^enabled=.*/enabled=0/g' /etc/dnf/plugins/subscription-manager.conf \
+    && dnf config-manager --disable \
+    && dnf update -y  && rm -rf /var/cache/dnf \
+    && dnf -y update-minimal --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs \
+    && echo "Installing new packages" \
+    && dnf -y --setopt=tsflags=nodocs install java-11-openjdk apr openssl &> /dev/null \
```

Comment:
We don't need `apr` and `openssl` packages when using the `netty-tcnative-boringssl-static` lib.

Reply:
Removed.